apps/aechannel/src/aesc_fsm.erl

```diff
@@ -1190,6 +1190,7 @@ handle_call(St, Req, From, #data{} = D) ->
     try handle_call_(St, Req, From, D)
     catch
         error:Error ->
+            lager:debug("CAUGHT ~p / ~p", [Error, erlang:get_stacktrace()]),
```

Comment:
True. There are a few places where this is done. It's needed mostly during testing, so this could be cleaned up pretty easily. [Tracked in Pivotal](https://www.pivotaltracker.com/story/show/164104947).